Dockerfile

```diff
@@ -1,41 +1,43 @@
-FROM php:7.2.12-cli-alpine3.8
-MAINTAINER Alejandro Celaya <alejandro@alejandrocelaya.com>
+FROM php:7.2.13-cli-alpine3.8
+LABEL maintainer="Alejandro Celaya <alejandro@alejandrocelaya.com>"
 
 ARG SHLINK_VERSION=1.15.0
 ENV SHLINK_VERSION ${SHLINK_VERSION}
+ENV APCu_VERSION 5.1.3
+ENV APCuBC_VERSION 1.0.3
 
 WORKDIR /etc/shlink
 
 RUN apk update && \
 
     # Install common php extensions
-    docker-php-ext-install -j$(nproc) pdo_mysql calendar && \
+    docker-php-ext-install -j"$(nproc)" pdo_mysql calendar && \
 
     # Install sqlite
     apk add --no-cache sqlite-libs sqlite-dev && \
-    docker-php-ext-install -j$(nproc) pdo_sqlite && \
+    docker-php-ext-install -j"$(nproc)" pdo_sqlite && \
 
     # Install other PHP packages that depend on other system packages
     apk add --no-cache icu-dev && \
-    docker-php-ext-install -j$(nproc) intl && \
+    docker-php-ext-install -j"$(nproc)" intl && \
 
     apk add --no-cache zlib-dev libpng-dev && \
-    docker-php-ext-install -j$(nproc) zip gd
+    docker-php-ext-install -j"$(nproc)" zip gd
 
 # Install APCu
-RUN wget https://pecl.php.net/get/apcu-5.1.3.tgz -O /tmp/apcu.tar.gz && \
+RUN wget "https://pecl.php.net/get/apcu-${APCu_VERSION}.tgz" -O /tmp/apcu.tar.gz && \
     mkdir -p /usr/src/php/ext/apcu && \
     tar xf /tmp/apcu.tar.gz -C /usr/src/php/ext/apcu --strip-components=1 && \
     docker-php-ext-configure apcu && \
-    docker-php-ext-install -j$(nproc) apcu && \
+    docker-php-ext-install -j"$(nproc)" apcu && \
     rm /tmp/apcu.tar.gz
 
 # Install APCu-BC extension
-RUN wget https://pecl.php.net/get/apcu_bc-1.0.3.tgz -O /tmp/apcu_bc.tar.gz && \
+RUN wget "https://pecl.php.net/get/apcu_bc-${APCuBC_VERSION}.tgz" -O /tmp/apcu_bc.tar.gz && \
     mkdir -p /usr/src/php/ext/apcu-bc && \
     tar xf /tmp/apcu_bc.tar.gz -C /usr/src/php/ext/apcu-bc --strip-components=1 && \
     docker-php-ext-configure apcu-bc && \
-    docker-php-ext-install -j$(nproc) apcu-bc && \
+    docker-php-ext-install -j"$(nproc)" apcu-bc && \
```

Comment:
What's the purpose of the `-j` flag? I've seen it used in some examples,but I don't know why.

Reply:
Enable parallel jobs during the installation process when it can.